sympy/integrals/transforms.py

```diff
@@ -1186,499 +1194,363 @@ def _laplace_build_rules(t, s):
+    (erf(sqrt(a/t)/2), (1-exp(-sqrt(a*s)))/s,
+     re(a)>0, S.Zero, dco), # 4.12.6
+    (erfc(sqrt(a*t)), (sqrt(s+a)-sqrt(a))/sqrt(s+a)/s,
+     S.true, -re(a), dco), # 4.12.9
+    (exp(a*t)*erfc(sqrt(a*t)), 1/(s+sqrt(a*s)),
+     S.true, S.Zero, dco), # 4.12.10
+    (erfc(sqrt(a/t)/2), exp(-sqrt(a*s))/s,
+     re(a)>0, S.Zero, dco), # 4.2.11
+    (besselj(n, a*t), a**n/(sqrt(s**2+a**2)*(s+sqrt(s**2+a**2))**n),
+     re(n)>-1, Abs(im(a)), dco), # 4.14.1
     (t**b*besselj(n, a*t),
      2**n/sqrt(pi)*gamma(n+S.Half)*a**n*(s**2+a**2)**(-n-S.Half),
-     And(And(a>0, n>-S.Half), Eq(b, n)), S.Zero, dco),
-    # 8.5
+     And(re(n)>-S.Half, Eq(b, n)), Abs(im(a)), dco), # 4.14.7
     (t**b*besselj(n, a*t),
      2**(n+1)/sqrt(pi)*gamma(n+S(3)/2)*a**n*s*(s**2+a**2)**(-n-S(3)/2),
-     And(And(a>0, n>-1), Eq(b, n+1)), S.Zero, dco),
-    # 8.6
-    (besselj(0, 2*sqrt(a*t)),
-     exp(-a/s)/s,
-     a>0, S.Zero, dco),
-    # 8.7, 8.8
-    (t**(b)*besselj(n, 2*sqrt(a*t)),
-     a**(n/2)*s**(-n-1)*exp(-a/s),
-     And(And(a>0, n>-1), Eq(b, n*S.Half)), S.Zero, dco),
-    # 8.9
-    (besselj(0, a*sqrt(t**2+b*t)),
-     exp(b*s-b*sqrt(s**2+a**2))/sqrt(s**2+a**2),
-     b>0, S.Zero, dco),
-    # 8.10, 8.11
-    (besseli(n, a*t),
-     a**n/(sqrt(s**2-a**2)*(s+sqrt(s**2-a**2))**n),
-     And(a>0, n>-1), Abs(a), dco),
-    # 8.12
+     And(re(n)>-1, Eq(b, n+1)), Abs(im(a)), dco), # 4.14.8
+    (besselj(0, 2*sqrt(a*t)), exp(-a/s)/s,
+     S.true, S.Zero, dco), # 4.14.25
+    (t**(b)*besselj(n, 2*sqrt(a*t)), a**(n/2)*s**(-n-1)*exp(-a/s),
+     And(re(n)>-1, Eq(b, n*S.Half)), S.Zero, dco), # 4.14.30
+    (besselj(0, a*sqrt(t**2+b*t)), exp(b*s-b*sqrt(s**2+a**2))/sqrt(s**2+a**2),
+     Abs(arg(b))<pi, Abs(im(a)), dco), # 4.15.19
+    (besseli(n, a*t), a**n/(sqrt(s**2-a**2)*(s+sqrt(s**2-a**2))**n),
+     re(n)>-1, Abs(re(a)), dco), # 4.16.1
     (t**b*besseli(n, a*t),
      2**n/sqrt(pi)*gamma(n+S.Half)*a**n*(s**2-a**2)**(-n-S.Half),
-     And(And(a>0, n>-S.Half), Eq(b, n)), Abs(a), dco),
-    # 8.13
+     And(re(n)>-S.Half, Eq(b, n)), Abs(re(a)), dco), # 4.16.6
     (t**b*besseli(n, a*t),
      2**(n+1)/sqrt(pi)*gamma(n+S(3)/2)*a**n*s*(s**2-a**2)**(-n-S(3)/2),
-     And(And(a>0, n>-1), Eq(b, n+1)), Abs(a), dco),
-    # 8.15, 8.16
-    (t**(b)*besseli(n, 2*sqrt(a*t)),
-     a**(n/2)*s**(-n-1)*exp(a/s),
-     And(And(a>0, n>-1), Eq(b, n*S.Half)), S.Zero, dco),
-    # 8.17
-    (bessely(0, a*t),
-     -2/pi*asinh(s/a)/sqrt(s**2+a**2),
-     a>0, S.Zero, dco),
+     And(re(n)>-1, Eq(b, n+1)), Abs(re(a)), dco), # 4.16.7
+    (t**(b)*besseli(n, 2*sqrt(a*t)), a**(n/2)*s**(-n-1)*exp(a/s),
+     And(re(n)>-1, Eq(b, n*S.Half)), S.Zero, dco), # 4.16.18
+    (bessely(0, a*t), -2/pi*asinh(s/a)/sqrt(s**2+a**2),
+     S.true, Abs(im(a)), dco), # 4.15.44
     # 8.18
-    (besselk(0, a*t),
-     (log(s+sqrt(s**2-a**2)))/(sqrt(s**2-a**2)),
-     a>0, Abs(a), dco)
+    (besselk(0, a*t), (log(s+sqrt(s**2-a**2)))/(sqrt(s**2-a**2)),
+     S.true, -re(a), dco) # 4.16.23
```

Comment:
This is incorrect, the result should be `(log( (s + sqrt(s**2-a**2))/a )/(sqrt(s**2-a**2))` according to Bateman54, p. 197.  The convergence plane is OK, though.

Reply:
Change (F) explained in comment below.